Orange/widgets/regression/owsgdregression.py

```diff
@@ -42,7 +40,7 @@ class OWSGDRegression(OWProvidesLearner, widget.OWWidget):
     learning_rate = settings.Setting(InvScaling)
 
     want_main_area = False
-    resizing_enabled = False
+    resizing_enabled = True
```

Comment:
but how i have try to implement it as another report like [owsvmclassification.py](https://github.com/biolab/orange3/blob/master/Orange/widgets/classify/owsvmclassification.py)
.